httpx/_utils.py

```diff
@@ -25,6 +25,52 @@
     r"|".join([re.escape(c) for c in _HTML5_FORM_ENCODING_REPLACEMENTS.keys()])
 )
 
+# For our supported text codecs, we start with the text codecs as supported by Chromium, Oct. 2023.
+# https://chromium.googlesource.com/chromium/chromium/+/refs/heads/trunk/chrome/browser/character_encoding.cc#36
+#
+# Then limit them to only includec codecs which are documented as included by cpython.
+# https://docs.python.org/3/library/codecs.html#standard-encodings
+#
+# We're referencing them with the canonical name as used by the Python codecs.
+# The alias given in the chromium source is included as a comment for comparison.
```

Comment:
> We're referencing them with the canonical name as used by the Python codecs.

* Will the canonical name for the codec be consistent across different Python implementations?
* Should we instead just be including all the possible aliases explicitly?
* What set of aliases does chromium support for these charset names?